core/src/test/java/org/elasticsearch/index/query/FuzzyQueryBuilderTests.java

```diff
@@ -143,6 +144,63 @@ public void testToQueryWithStringFieldDefinedFuzziness() throws IOException {
         assertThat(fuzzyQuery.getPrefixLength(), equalTo(1));
     }
 
+    public void testToQueryWithStringFieldDefinedWrongFuzziness() throws IOException {
+        assumeTrue("test runs only when at least a type is registered", getCurrentTypes().length > 0);
```

Comment:
Instead of query1, query2, and query3,  I believe you should rename the queries with the goal that you are trying to accomplish with each one.  I believe that I should be able to know what the variable means without having to look at the entire test to do so.